src/components/delegateRegistration/summary/summary.js

```diff
@@ -1,28 +1,79 @@
 import React from 'react';
 import AccountVisual from '../../accountVisual/index';
 import { fromRawLsk } from '../../../utils/lsk';
+import PassphraseInputV2 from '../../passphraseInputV2/passphraseInputV2';
+import { PrimaryButtonV2, TertiaryButtonV2 } from '../../toolbox/buttons/button';
+import { extractPublicKey } from '../../../utils/account';
 import Fees from '../../../constants/fees';
-import TransactionSummary from '../../transactionSummary';
```

Comment:
Why did you remove TransactionSummary from this delegate registration summary component? delegate registration is a transaction type and it should use the general TransactionSummary component as we talk about it.

Reply:
that was causing a problem related the second passphrase, because of that component the second passphrase was being displayed in the middle of the component above of Fee.

Reply:
<img width="1475" alt="Screenshot 2019-07-08 at 12 27 13" src="https://user-images.githubusercontent.com/6761937/60804027-d69ada00-a17c-11e9-83a6-0069f930a874.png">
the design of the delegate summary isn’t aligned with the latest design. The fee should always be at the top of second passphrase inputs. So, please change the transaction summary to show the fee at the top of second passphrase input and use that instead of your custom summary component.

Reply:
Yashar I think is better to refactor the TransationSummary component, right now the delegate Summary is exactly the same as the Send Summary

after refactor the TransactionSummary component to looks exactly as should be in all pages I can replace it in Send/Summary and DelegateReg/Summarywill be easier

Reply:
It doesn't require refactoring at all. You just need to rearrange some elements inside the render method to be aligned with the design. You must move the fee section to the top of the passphrase input component.